ERROR getway: 2020/11/06 23:37:59 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:38:40 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:39:16 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:40:02 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:41:03 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:42:14 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:44:06 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:44:28 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/06 23:52:42 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:00:54 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:03:27 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:03:59 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:15:04 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:17:46 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:17:55 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:19:20 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:21:26 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:22:59 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:25:34 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:26:40 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:28:43 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:30:44 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:32:10 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:32:46 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:33:20 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:36:20 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:39:31 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:40:47 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:41:14 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 00:42:31 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 15:12:14 listen tcp :8081: bind: address already in use
ERROR user  : 2020/11/07 15:12:15 failed: pq: no pg_hba.conf entry for host "176.59.34.90", user "phill", database "ap_user_db", SSL on
ERROR getway: 2020/11/07 15:12:34 listen tcp :8081: bind: address already in use
ERROR user  : 2020/11/07 15:12:34 failed: pq: no pg_hba.conf entry for host "176.59.34.90", user "phill", database "ap_user_db", SSL on
ERROR user  : 2020/11/07 15:13:36 failed: pq: no pg_hba.conf entry for host "176.59.34.90", user "phill", database "ap_user_db", SSL on
ERROR user  : 2020/11/07 15:16:49 failed: pq: no pg_hba.conf entry for host "176.59.34.90", user "phill", database "ap_user_db", SSL on
ERROR getway: 2020/11/07 16:04:34 listen tcp :8081: bind: address already in use
ERROR getway: 2020/11/07 16:07:23 listen tcp :8081: bind: address already in use
ERROR user  : 2020/11/07 19:06:12 failed: pq: no pg_hba.conf entry for host "109.252.192.145", user "phill", database "ap_user_db", SSL on
ERROR user  : 2020/11/07 20:18:50 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/07 20:25:58 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/07 20:32:31 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/07 20:32:55 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/07 20:41:16 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/07 20:44:50 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/07 20:45:07 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/07 20:45:27 failed: pq: password authentication failed for user "phill1"
ERROR user  : 2020/11/09 11:43:13 record not found
ERROR user  : 2020/11/09 11:43:22 record not found
ERROR user  : 2020/11/09 11:45:43 record not found
ERROR user  : 2020/11/09 11:52:57 [101 121 74 104 98 71 99 105 79 105 74 73 85 122 73 49 78 105 73 115 73 110 82 53 99 67 73 54 73 107 112 88 86 67 74 57 46 101 121 74 108 101 72 65 105 79 106 69 50 77 68 81 53 77 84 73 49 78 68 65 115 73 110 86 122 90 88 74 102 97 87 81 105 79 105 74 81 97 71 108 115 97 88 66 119 73 70 112 111 100 87 120 108 100 105 73 115 73 110 86 49 97 87 81 105 79 105 73 120 77 106 78 108 78 68 85 50 78 121 49 108 79 68 108 105 76 84 69 121 90 68 77 116 89 84 81 49 78 105 48 48 77 106 89 50 78 84 85 48 78 68 65 119 77 68 65 105 102 81 46 80 121 88 72 107 117 50 114 108 107 113 112 84 77 49 105 105 113 69 105 83 89 102 108 115 105 77 113 74 76 103 99 112 82 48 68 53 74 122 90 50 68 119]
ERROR user  : 2020/11/09 11:53:33 eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2MDQ5MTI1NDAsInVzZXJfaWQiOiJQaGlsaXBwIFpodWxldiIsInV1aWQiOiIxMjNlNDU2Ny1lODliLTEyZDMtYTQ1Ni00MjY2NTU0NDAwMDAifQ.PyXHku2rlkqpTM1iiqEiSYflsiMqJLgcpR0D5JzZ2Dw
ERROR user  : 2020/11/09 12:04:46 record not found
ERROR user  : 2020/11/09 12:05:02 record not found
ERROR user  : 2020/11/09 12:08:47 eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2MDQ5MTM5MTgsInVzZXJfaWQiOiJQaGlsaXBwIFpodWxldiIsInV1aWQiOiIxMjNlNDU2Ny1lODliLTEyZDMtYTQ1Ni00MjY2NTU0NDAwMDAifQ.ZnmzfIUk-
ERROR user  : 2020/11/09 12:10:18 eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2MDQ5MTM5OTQsInVzZXJfaWQiOiJQaGlsaXBwIFpodWxldiIsInV1aWQiOiIxMjNlNDU2Ny1lODliLTEyZDMtYTQ1Ni00MjY2NTU0NDAwMDAifQ.PGByK
ERROR user  : 2020/11/09 12:12:57 eyJleHAiOjE2MDQ5MTQxNzUsInVzZXJfaWQiOiJQaGlsaXBwIFpodWxldiIsInV1aWQiOiIxMjNlNDU2Ny1lODliLTEyZDMtYTQ1Ni00MjY2NTU0NDAwMDAifQ
ERROR user  : 2020/11/09 12:26:29 failed: pq: no pg_hba.conf entry for host "194.186.207.223", user "phill", database "ap_user_db", SSL on
ERROR user  : panic: runtime error: index out of range [2] with length 0
ERROR user  : 
ERROR user  : goroutine 6 [running]:
ERROR user  : encoding/base64.(*Encoding).decodeQuantum(0xc0000da000, 0x0, 0x0, 0x0, 0xc000024380, 0x7a, 0x80, 0x4, 0xc000024380, 0x80, ...)
ERROR user  : 	/usr/local/go/src/encoding/base64/base64.go:362 +0x549
ERROR user  : encoding/base64.(*Encoding).Decode(0xc0000da000, 0x0, 0x0, 0x0, 0xc000024380, 0x7a, 0x80, 0x0, 0x3, 0xc00007e480)
ERROR user  : 	/usr/local/go/src/encoding/base64/base64.go:530 +0x5fc
ERROR user  : task/provider/services/user/internal/app/handler.Init.HandleUserGet.func1(0xc00007e480, 0xc0001ea900, 0x156a780, 0xc00007e480, 0x0, 0x0, 0x38)
ERROR user  : 	/Users/u16837531/Documents/micro_services/services/user/internal/app/handler/handleGet.go:26 +0x185
ERROR user  : task/provider/services/user/internal/app/handler.Init.HandleUserGet(0xc00007e480, 0xc0001ea900, 0x1675a40, 0x0, 0x0, 0xc000138100, 0x0)
ERROR user  : 	/Users/u16837531/Documents/micro_services/services/user/internal/app/handler/handleGet.go:45 +0x35
ERROR user  : task/provider/services/user/internal/app/server.(*initServer).UserGet(0xc00022a5a0, 0x1673e40, 0xc00010ca80, 0xc00007e480, 0xc00022a5a0, 0xc00010ca80, 0xc000066ba0)
ERROR user  : 	/Users/u16837531/Documents/micro_services/services/user/internal/app/server/server.go:107 +0x3f
ERROR user  : task/provider/proto/user._UserService_UserGet_Handler(0x1563580, 0xc00022a5a0, 0x1673e40, 0xc00010ca80, 0xc0000346c0, 0x0, 0x1673e40, 0xc00010ca80, 0xc00013e1a0, 0xce)
ERROR user  : 	/Users/u16837531/Documents/micro_services/proto/user/user_grpc.pb.go:170 +0x214
ERROR user  : google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000ad6c0, 0x1678ba0, 0xc000083080, 0xc000138100, 0xc0001cd560, 0x1967da8, 0x0, 0x0, 0x0)
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:1210 +0x522
ERROR user  : google.golang.org/grpc.(*Server).handleStream(0xc0000ad6c0, 0x1678ba0, 0xc000083080, 0xc000138100, 0x0)
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:1533 +0xd05
ERROR user  : google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0000b5d60, 0xc0000ad6c0, 0x1678ba0, 0xc000083080, 0xc000138100)
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:871 +0xa5
ERROR user  : created by google.golang.org/grpc.(*Server).serveStreams.func1
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:869 +0x1fd
ERROR user  : panic: runtime error: index out of range [2] with length 0
ERROR user  : 
ERROR user  : goroutine 31 [running]:
ERROR user  : encoding/base64.(*Encoding).decodeQuantum(0xc0000da000, 0x0, 0x0, 0x0, 0xc0000d7d00, 0x7a, 0x80, 0x4, 0xc0000d7d00, 0x80, ...)
ERROR user  : 	/usr/local/go/src/encoding/base64/base64.go:362 +0x549
ERROR user  : encoding/base64.(*Encoding).Decode(0xc0000da000, 0x0, 0x0, 0x0, 0xc0000d7d00, 0x7a, 0x80, 0x0, 0x3, 0xc0000c71c0)
ERROR user  : 	/usr/local/go/src/encoding/base64/base64.go:530 +0x5fc
ERROR user  : task/provider/services/user/internal/app/handler.Init.HandleUserGet.func1(0xc0000c71c0, 0xc0001ea900, 0x156a780, 0xc0000c71c0, 0x0, 0x0, 0x38)
ERROR user  : 	/Users/u16837531/Documents/micro_services/services/user/internal/app/handler/handleGet.go:26 +0x185
ERROR user  : task/provider/services/user/internal/app/handler.Init.HandleUserGet(0xc0000c71c0, 0xc0001ea900, 0x1675a40, 0x0, 0x0, 0xc000202400, 0x0)
ERROR user  : 	/Users/u16837531/Documents/micro_services/services/user/internal/app/handler/handleGet.go:45 +0x35
ERROR user  : task/provider/services/user/internal/app/server.(*initServer).UserGet(0xc00022a5c0, 0x1673e40, 0xc000262210, 0xc0000c71c0, 0xc00022a5c0, 0xc000262210, 0xc000069ba0)
ERROR user  : 	/Users/u16837531/Documents/micro_services/services/user/internal/app/server/server.go:107 +0x3f
ERROR user  : task/provider/proto/user._UserService_UserGet_Handler(0x1563580, 0xc00022a5c0, 0x1673e40, 0xc000262210, 0xc000223800, 0x0, 0x1673e40, 0xc000262210, 0xc000221d40, 0xce)
ERROR user  : 	/Users/u16837531/Documents/micro_services/proto/user/user_grpc.pb.go:170 +0x214
ERROR user  : google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000ad6c0, 0x1678ba0, 0xc000083500, 0xc000202400, 0xc0001cd560, 0x1967da8, 0x0, 0x0, 0x0)
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:1210 +0x522
ERROR user  : google.golang.org/grpc.(*Server).handleStream(0xc0000ad6c0, 0x1678ba0, 0xc000083500, 0xc000202400, 0x0)
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:1533 +0xd05
ERROR user  : google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0000b5d90, 0xc0000ad6c0, 0x1678ba0, 0xc000083500, 0xc000202400)
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:871 +0xa5
ERROR user  : created by google.golang.org/grpc.(*Server).serveStreams.func1
ERROR user  : 	/Users/u16837531/go/pkg/mod/google.golang.org/grpc@v1.33.1/server.go:869 +0x1fd
ERROR getway: 2020/11/13 12:47:10 listen tcp :8081: bind: address already in use
